src/controllers/accounts/AccountsBalanceInfoController.ts

```diff
@@ -68,7 +68,7 @@ export default class AccountsBalanceController extends AbstractController<Accoun
 		const tokenArg =
 			typeof token === 'string'
 				? token.toUpperCase()
-				: this.api.registry.chainToken;
+				: this.api.registry.chainTokens[0];
```

Comment:
Don't we need to `.toUpperCase()` this one too?

Also: a comment documenting the assumption that `[0]` is the native token would be good?

Reply:
This is a good point. AFAIK there is nothing normalizng the token symbols to be uppercase, I just took it as convention. Changing it to always force upper case